dev-docs/bidders/seedingAlliance.md

```diff
@@ -1,11 +1,25 @@
 ---
 layout: bidder
-title: SeedingAlliance
-description: Prebid SeedingAlliance Bidder Adaptor
-pbjs: true
+title: Seeding Alliance
+description: Prebid Seeding Alliance Bidder Adapter
 biddercode: seedingAlliance
-media_types: native
 gdpr_supported: true
+gvl_id: 371 
+usp_supported: false
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+userId: none
+media_types: banner, native
+safeframes_ok: true
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
+pbs_app_supported: false
+prebid_member: false
+multiformat_supported: check with bidder
```

Comment:
If you don't know if you support it, then you don't need that entry.

I would recommend setting it to one of the three possible values.

Reply:
I just removed it - thanks for the hint!